src/content/language/constructors.md

```diff
@@ -318,50 +272,34 @@ class Point {
 }
 ```
 
+### Factory constructors
 
-## Constant constructors
-
-If your class produces objects that never change, you can make these
-objects compile-time constants. To do this, define a `const` constructor
-and make sure that all instance variables are `final`.
-
-<?code-excerpt "misc/lib/language_tour/classes/immutable_point.dart"?>
-```dart
-class ImmutablePoint {
-  static const ImmutablePoint origin = ImmutablePoint(0, 0);
-
-  final double x, y;
-
-  const ImmutablePoint(this.x, this.y);
-}
-```
-
-Constant constructors don't always create constants.
-For details, see the section on
-[using constructors][].
-
+When encountering one of two cases of implementing a constructor,
+use the `factory` keyword:
```

Comment:
[nit] Is this missing a word at the end? `use the factory keyword when:` ? `use the factory keyword if:` ?